website/routes.py

```diff
@@ -882,6 +882,24 @@ def make_url_map(app):
             project_views.register.node_registration_embargo_disapprove,
             OsfWebRenderer('error.mako', trust=False)
         ),
+        Rule(
```

Comment:
seems fine. One thing we can look at in the future is a generic token system, where the action is not embedded. Instead, the token would include the information as to what is being done. For instance, in this case, if I change the key approval to disapproval, would that token then disapprove?